packages/Popover/src/components/Trigger/Trigger.js

```diff
@@ -41,10 +48,9 @@ function Trigger(props) {
   const { children, a11yText } = props;
 
   if (typeof children === "function") {
-    return <React.Fragment>{children(handleTriggerEvent)}</React.Fragment>;
+    return <React.Fragment>{children(handleTriggerEvent, content.ariaId)}</React.Fragment>;
```

Comment:
What if you cloned this `children` element and added the `id` to its props, so that the consumer never even needed to know it existed?

I used this technique for the `<DropdownMenu>`:
https://github.com/acl-services/paprika/blob/master/packages/DropdownMenu/src/DropdownMenu.js#L74